src/benchmark.cc

```diff
@@ -336,6 +340,31 @@ ConsoleReporter::OutputOptions GetOutputOptions(bool force_no_color) {
 
 }  // end namespace internal
 
+BenchmarkTime::BenchmarkTime()
+    : to_cost_in_seconds_(SecondsCost),
+      cpu_unit_multiplier_(TimeUnit::kNanosecond),
+      unit_multiplier_(TimeUnit::kNanosecond),
+      unit_name_("s"),
```

Comment:
we're defaulting to "s" but with nanosecond multipliers? that might warrant a comment to explain it.

Reply:
I added a comment!